Python/GreatestCommonDivisor.py

```diff
@@ -0,0 +1,14 @@
+def (a,b): 

```

Comment:
@tanishq67 I believe it was missing to declare the name of the function, could you correct this part?